src/GraphQL/Types/Composite/ObjectGraphType.cs

```diff
@@ -7,8 +7,14 @@ namespace GraphQL.Types
     /// </summary>
     public interface IObjectGraphType : IComplexGraphType, IImplementInterfaces
     {
+        /// <summary>
+        /// Gets or sets a delegate that determines if the specified object is valid for this graph type.
+        /// </summary>
         Func<object, bool> IsTypeOf { get; set; }
 
+        /// <summary>
+        /// Adds a ..........???
```

Comment:
Help !  What does this do @sungam3r  ?

Reply:
Adds an instanse of \<see cref="IInterfaceGraphType"/> to the list of interface instances supported by this object graph type.

It's like `ISchema.AdditionalTypeInstances`.

Reply:
```suggestion
        /// Adds an instanse of <see cref="IInterfaceGraphType"/> to the list of interface instances supported by this object graph type.
```